<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DataWidget</class>
 <widget class="QWidget" name="DataWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QComboBox" name="DataSel"/>
   </item>
   <item>
    <widget class="QStackedWidget" name="DataStack">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="NoDataPage">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="NoDataLabel">
         <property name="text">
          <string>No Data selected</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="Data2DWidget" name="TwoDWidget"/>
     <widget class="Data3DWidget" name="ThreeDWidget"/>
     <widget class="Data3DVWidget" name="VecWidget"/>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Data2DWidget</class>
   <extends>QWidget</extends>
   <header>data/data2dwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Data3DWidget</class>
   <extends>QWidget</extends>
   <header>data/data3dwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Data3DVWidget</class>
   <extends>QWidget</extends>
   <header>data/data3dvwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
